files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml

```diff
@@ -150,8 +150,10 @@ paralyzer:
 
 cuffmerge: {mem: 8}
 bio3d_pca: {mem: 64}
-bio3d_rmsd: {mem: 64}
-bio3d_rmsf: {mem: 64}
+bio3d_rmsd: {mem: 100}
+bio3d_rmsf: {mem: 100}
+rxdock_sort_filter: {mem: 100}
```

Comment:
I haven't see rdock in this list. Are you sure rdock needs that many memory? We have been running 1000 of jobs without problems and just providing 4GB I think.

Reply:
I was talking about these `sort_filter` yesterday in the chat.